pkg/branding/env.go

```diff
@@ -0,0 +1,21 @@
+package branding
+
+import "github.com/stackrox/rox/pkg/env"
+
+const (
+	// ProductBrandingEnvName is the name of environment variable that defines product branding: commercial or open source.
+	ProductBrandingEnvName = "ROX_PRODUCT_BRANDING"
+
+	// ProductBrandingRHACS is the value ProductBrandingEnvName should have for Red Hat Advanced Cluster Security branded builds.
+	ProductBrandingRHACS = "RHACS_BRANDING"
+)
+
+var (
+	// TODO @jschnath: Remove the default in the followup task of adding the new env variable to CI
```

Comment:
Same here

Reply:
Created the followup task and linked it in the TODO